### PR TITLE
disable harry-integration local (in-jvm) module for standalone mode

### PR DESCRIPTION
**Pull request by fruch**
In case we don't need the docker integration of `harry-integration`
and we are using an external running cassandra

Signed-off-by: Amos Kong <amos@scylladb.com>

### CODE REVIEW

**Comment by roydahan** [comment]
@ifesdjeen could you please review?

@fruch you may need to open a JIRA ticket so they'll see this PR.

File: conf/external.yaml [hunk 1]
```diff
@@ -92,6 +89,9 @@ clustering_descriptor_selector:
 # and model state.
 runner:
   sequential:
+    run_time: 10
+    run_time_unit: "MINUTES"
```
Comment (roydahan) [inline conf/external.yaml]: @fruch did you intend to shorten it from 2 hours to 10 minutes?

Comment (fruch) [reply conf/external.yaml]: no, I copy pasted for a different place, anyhow I think it's an example configuration.
But I'll fix that

Comment (fruch) [reply conf/external.yaml]: fixed

**Comment by fruch** [comment]
> @ifesdjeen could you please review?
> 
> @fruch you may need to open a JIRA ticket so they'll see this PR.

Now I've opened a JIRA: https://issues.apache.org/jira/browse/CASSANDRA-17261

**Pull request merged**